e2e-tests/portal/cypress/e2e/journeys/admin/restrict-access/restrict-access.spec.js

```diff
@@ -13,17 +12,13 @@ context('Only allow authorised users to access admin pages', () => {
     let deal;
 
     before(() => {
-      const aDealInStatus = (status) => deals.filter((aDeal) => status === aDeal.status)[0];
-
       cy.deleteDeals(ADMIN);
-      cy.insertOneDeal(aDealInStatus('Draft'), BANK1_MAKER1).then((insertedDeal) => {
-        deal = insertedDeal;
-      });
+      cy.createBssDeal({});
     });
 
     it('allows read only user with all bank access to view deal', () => {
       cy.loginGoToDealPage(ADMIN, deal);
-      cy.url().should('eq', relative(`/contract/${deal._id}`));
+      cy.url().should('eq', relative(`/dashboard/deals/0`));
```

Comment:
Need to update this test - it was previously testing the contract/deal URL, but now its testing the dashboard URL